src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -30,10 +29,12 @@
 public class UPSTrackingJob implements Job {
```

Comment:
Maybe rename to `TestBostonUPSTrackingJob`?  There's are a few assumptions in this code that apply only to testboston, so calling this out via the class name will alert the reader to the fact that this is not a general purpose UPS tracking utility.

Reply:
Alternatively, we could fully parameterize `SQL_SELECTION_KITS` in the constructor so that the label matches and instance name can be controlled by clients.